src/abstract-wallet.ts

```diff
@@ -637,6 +637,12 @@ export abstract class AbstractWallet {
     async isOnchainAuthSigningKeySet(nonce: Nonce = 'committed'): Promise<boolean> {
         const mainRifRollupContract = this.getRifRollupMainContract();
```

Comment:
Could we move this line after the nonce check?

Reply:
sure!